Add banned field to SafeContract model (#1400)
safe-global/safe-transaction-service: README.md

### What chains do you officially support?
https://docs.safe.global/learn/safe-core/safe-core-api/available-services

### What means banned field in SafeContract model?

> The 'banned' field in the 'SafeContract' model is used to prevent indexing of certain safes that have unsupported MasterCopy and unverified contracts that have issues during indexing. This field does not remove the banned safe and indexing can be resumed once the issue has been resolved.

Reviewer: ```suggestion
The `banned` field in the `SafeContract` model is used to prevent indexing of certain Safes that have an unsupported `MasterCopy` or unverified proxies that have issues during indexing. This field does not remove the banned Safe and indexing can be resumed once the issue has been resolved.
```
